applications/dashboard/models/class.activitymodel.php

```diff
@@ -879,17 +883,24 @@ public function sendNotification($ActivityID, $Story = '', $Force = false) {
                     // Only send if the user is not banned
                     if (!val('Banned', $User)) {
                         $Email->send();
+                        $Emailed = self::SENT_OK;
```

Comment:
That should already be the case here:

```php
// Only send if the user is not banned
if (!val('Banned', $User)) {
    $Email->send();
    $Emailed = self::SENT_OK;
} else {
    $Emailed = self::SENT_SKIPPED;
}
```

If `User.Banned` is truthy, it should avoid even trying to send the message and immediately go to a status of `ActivityModel::SENT_SKIPPED`.